edx_proctoring/templates/proctored_exam/exam_not_submitted.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have not yet submitted this proctored exam
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Make sure you return to the proctoring software and select <strong> Quit </strong> to end proctoring
+      and submit your exam.
+    {% endblocktrans %}
+  </p>
+
+  <p>
+    {% blocktrans %}
+      If you have questions about the status of your proctored exam results, please contact {{ platform_name }} Support.
```

Comment:
Strike "please": 
"...status of your proctored exam results, contact ``{{ platform_name }} Support."